docs/_docs/variables.md

```diff
@@ -193,9 +193,14 @@ following is a reference of the available data.
         <code>_config.yml</code> are available through the <code>site</code>
         variable. For example, if you have <code>url: http://mysite.com</code>
         in your configuration file, then in your Posts and Pages it will be
-        stored in <code>site.url</code>. Jekyll does not parse changes to
-        <code>_config.yml</code> in <code>watch</code> mode, you must restart
-        Jekyll to see changes to variables.
+        stored in <code>site.url</code>. But there is
```

Comment:
Perhaps it's the right time to add this in a dedicated `site.url` row in this table to mention differences between development and production environments?